nextcloudappstore/core/models.py

```diff
@@ -106,6 +106,7 @@ class Meta:
         verbose_name = _('App Release')
         verbose_name_plural = _('App Releases')
         unique_together = (('app', 'version'),)
+        ordering = ['-version']
```

Comment:
I misunderstood you. I thought you meant semantic versioning also uses natural ordering.
If it doesn't then this won't work.

I just saw that semantic_version has django fields. Maybe we should use them, then we can at least use sort() to get the correct order.

It even has a SpecField which could be used to replace the min_* and max_*  fields. https://pypi.python.org/pypi/semantic_version/